bundles/best_practices.rst

```diff
@@ -166,6 +166,69 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
```

Comment:
I would put a few other recommendations for their CI setup:

- have a dev requirement on the `symfony/phpunit-bridge "^3.3 || ^4"` and run it in strict mode (which is the default), so that any usage of a deprecated API is catched. The only job where I'm allowing weak mode is the lowest job (because then, a version of your dependency may trigger a deprecation just because it went older than the release using the new API, which might be very hard to avoid in this job if it is a transitive dependency)
- recommend enabling a Travis cron if your repository is not very active, so that failures on the unstable jobs due to an upstream change have a chance to occur before this change becomes stable. If the repo is very active, this is useless (because some activity-related build will also catch it), but most bundles are far less active than Symfony and will benefit from it (I generally use a monthly cron, but you could make it a weekly one instead)

Reply:
sounds like an excellent idea to me (and was not aware thats possible). often, i notice such problems when somebody does a small contribution or even provides a trivial typo fix, and then wonder why that makes the build break. so far i manually restarted master to be sure, but that would automate that away and we would be aware of problems earlier. 

weekly sounds like a good rate to me.